brownie/project/compiler/__init__.py

```diff
@@ -14,9 +14,7 @@
 from brownie.exceptions import UnsupportedLanguage
 from brownie.project import sources
 from brownie.project.compiler.solidity import (  # NOQA: F401
-    find_best_solc_version,
     find_solc_versions,
-    install_solc,
```

Comment:
removing these imports is breaking other functionality

Reply:
sorry, it was my mistake, fixed